test/components/PrintLabelsRouter.spec.js

```diff
@@ -19,15 +19,15 @@ describe('PrintLabelsRouter.vue', () => {
   beforeEach(() => {
     wrapper = mount(PrintLabelsRouter, {
       localVue,
-      stubs: ['nuxt-link']
+      stubs: ['nuxt-link'],
     })
   })
 
   it('is a Vue instance', () => {
     expect(wrapper.findComponent(PrintLabelsRouter).exists()).toBeTruthy()
   })
 
-  each(links).it('will have a link to %s', (link) => {
+  each(links).test('will have a link to %s', (link) => {
```

Comment:
just of interest, what was the issue with it()?

Reply:
Eslint was complaining about something so this was to appease it!